client/web/reporter/src/components/Common/Prefilter.vue

```diff
@@ -181,6 +187,7 @@
     >
       <b-button
         variant="primary"
+        size="sm"
```

Comment:
Could we used the wrapper button for this?

Reply:
1. It won't be intuitive because the wrapper button specifically is to add table items. 
2. I guess we could make the v-else work, but it ties back to not being intuitive for future work. 
3. The benefit of reducing template clutter, at the expense of having to go to lib/vue to figure out what is going on with that button does not feel much